pkg/commands/drift/statefiles/statefiles.go

```diff
@@ -35,18 +37,32 @@ import (
 
 var _ cli.Command = (*DriftStatefilesCommand)(nil)
 
+const (
+	issueTitle = "Terraform statefile drift detected"
+	issueBody  = `We've detected a drift between the statefiles stored in your GCS bucket and the
+        ones referenced in your backend config blocks.
+
+        See the comment(s) below to see details of the drift
+
+        Please determine which parts are correct, and delete or rename any unused statefiles
+        in your GCS bucket.
+
+        Re-run drift detection manually once complete to verify all diffs are properly resolved.`
+)
+
 type DriftStatefilesCommand struct {
 	cli.BaseCommand
 
 	directory string
 
 	flags.GitHubFlags
 	flags.RetryFlags
+	driftflags.DriftIssueFlags
 
 	flagGCSBucket string
 
-	githubClient    github.GitHub
 	terraformParser parser.Terraform
+	issueService    drift.GitHubDriftIssueService
```

Comment:
This should be a pointer since it's a struct

Reply:
Done